source/velocity_boundary_conditions/gplates.cc

```diff
@@ -642,16 +642,17 @@ namespace aspect
     void
     GPlates<dim>::update ()
     {
-      if (time_dependent && (this->get_time() - first_data_file_model_time >= 0.0))
+      const double time_since_start = this->get_time() - first_data_file_model_time;
+
+      if (time_dependent && (time_since_start >= 0.0))
         {
-          const double time_steps_since_start = (this->get_time() - first_data_file_model_time)
-                                                / data_file_time_step;
+          const int time_steps_since_start = static_cast<int> (time_since_start / data_file_time_step);
```

Comment:
And the rounding to zero is intended in this case. I need the file number before the current time.